File: test/fixtures/output/javascript/axios/unparseable-query.js
import axios from "axios";

const options = {method: 'GET', url: 'http://mockbin.com/har?&&a=b&&'};

axios.request(options).then(function (response) {
  console.log(response.data);
}).catch(function (error) {
  console.error(error);
});

File: test/fixtures/output/java/nethttp/unparseable-query.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://mockbin.com/har?&&a=b&&"))
    .method("GET", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());